src/pykeen/models/multimodal/combinations.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of combinations for the LiteralE model."""
+
+import torch
+from torch import nn as nn
+
+from ...utils import combine_complex, split_complex
+
+__all__ = [
+    'DistMultCombination',
+    'ComplExLiteralCombination',
+]
+
+
+class DistMultCombination(nn.Sequential):
+    """Transform the embeddings and the literals together."""
+
+    def __init__(
+        self,
+        embedding_dim: int,
+        num_of_literals: int,
+        input_dropout: float = 0.0,
+    ):
+        linear = nn.Linear(embedding_dim + num_of_literals, embedding_dim)
+        dropout = nn.Dropout(input_dropout)
```

Comment:
@mberr note the dropout comes after the linear later for distmult but before for complex. was this correct?